bootstrap/sql/migrations/native/1.5.9/mysql/schemaChanges.sql

```diff
@@ -0,0 +1,5 @@
+-- Extend app extension for limits
+ALTER TABLE apps_extension_time_series ADD COLUMN extension VARCHAR(255);
+UPDATE apps_extension_time_series SET extension = 'status' WHERE extension IS NULL;
```

Comment:
Just curious what the `extension = 'status'` represents?

Reply:
before we only used the `apps_extension_time_series` to store status information about the Internal Apps (e.g., Data Insights or Reindex).

Now however I re-purposed the table to also store:
- `limits` info about app usage
- `custom` state of the app that needs to be tracked.

The `status` just helps us index the data and fetch whatever is needed in each scenario